robot_calibration/include/robot_calibration/capture/feature_finder_loader.h

```diff
@@ -25,7 +25,7 @@
 #include <boost/shared_ptr.hpp>
 
 #include <ros/ros.h>
-#include <pluginlib/class_loader.h>
+#include <pluginlib/class_loader.hpp>
```

Comment:
any idea how far back this was added? Does it exist in melodic & noetic?

Reply:
I think somewhere before 2017, it is in melodic already.